srcpkgs/pipewire/template

```diff
@@ -16,14 +16,15 @@ configure_args="
  -Dbluez5-codec-lc3plus=disabled
  -Dsession-managers=[]
  -Drlimits-match=@_pipewire
+ -Dlibffado=$(vopt_if ffado enabled disabled)
```

Comment:
```suggestion
 $(vopt_bool ffado libffado)
```

Reply:
That option specifically looks for "enabled", "disabled", or "auto" (see error below), which I don't think any of the `vopt_` shortcuts provide.

`meson.build:1:0: ERROR: Value "false" (of type "string") for combo option "Enable code that depends on libffado" is not one of the choices. Possible choices are (as string): "enabled", "disabled", "auto".`

Reply:
hm strange, meson is usually good about that